### DISCUSSION TITLE
regarding Station Overview

### DISCUSSION THREAD

**Discussion by Vicinis** [AzuraCast/AzuraCast#5527] [Q&A] [answered]
I run several several stations, each on their on servers. And one server (seperate) for website.
is it possible to see the Station Overview of all stations on one page (some kind of code to put on my website) instead of going to each station?
Or any other ideas?
Thanks in advance

**Comment by BusterNeece** [comment, marked as answer]
@Vicinis It's not possible right now, but you could implement it via the API endpoints that we expose for the various statistics, or submit a feature request for us to build out a system-wide page for it: https://features.azuracast.com/

**Reply by Vicinis** [reply]
Thanks. That's the only disadvantage I have by running them on separate servers. But I still prefer it this way.
Cheers
